aten/src/ATen/core/EnableNamedTensor.h

```diff
@@ -1,2 +0,0 @@
-#pragma once
-// We are working on removing the BUILD_NAMEDTENSOR flag from the codebase.
```

Comment:
it's a bit unclear to me from github, is this removing the file or just the contents of the file?

Reply:
This is removing the file

Reply:
![image](https://user-images.githubusercontent.com/5652049/70651392-b3d68b00-1c1e-11ea-90d2-2b25895a1c72.png)